applications/app/controllers/CrosswordsController.scala

```diff
@@ -313,15 +313,19 @@ class CrosswordEditionsController(
     getCrosswords
       .map(parseCrosswords)
       .flatMap { crosswords =>
-        remoteRenderer.getEditionsCrossword(wsClient, crosswords)
+        crosswords.headOption match {
+          case Some(crossword) => remoteRenderer.getEditionsCrossword(wsClient, crossword)
+          case None            => Future.successful(NoContent)
+        }
       }
   }
 
   def digitalEditionJson: Action[AnyContent] = Action.async { implicit request =>
     getCrosswords
       .map(parseCrosswords)
       .map { crosswords =>
-        Cached(CacheTime.Default)(RevalidatableResult.Ok(toJson(crosswords))).as("application/json")
+        val responseJson = Json.obj("crosswords" -> crosswords)
```

Comment:
As mentioned above, it looks like the JSON response has a list of crosswords as before, but the page itself doesn't?